src/schemas/components/SchemaFormPropertyArray.vue

```diff
@@ -5,11 +5,12 @@
         No items in this list
       </p>
     </template>
-    <p-draggable-list v-model="value" allow-create allow-delete :generator="generator">
+    <p-draggable-list v-model="value" allow-create allow-delete :generator="generator" :state="state">
```

Comment:
draggable list doesn't accept `State` yet but this will future proof for that.

Reply:
Ah I wasn't really thinking of these as inputs since they don't extend the base input class but i see the use case - let's chat at some point to see what that should look like

Reply:
might not even be necessary. We'll figure it out if we need.